messageBundles/gpii-app-qss-settings_en.json

```diff
@@ -62,9 +62,9 @@
         "footerTip": "You can also use the “CTRL” key, plus SCROLL (wheel or gesture) to do Zoom Top Window without using Morphic."
     },
     "gpii_app_qss_settings_common-language": {
-        "tooltip": "<p>Change Windows language setting.</p><p>Works for menus, etc. in Windows, Edge, and many other applications that have Language Packs installed.</p>",
-        "title": "Language",
-        "tip": "<p>Changes Language of Windows</p><p>Also works for apps that support Windows language settings.</p><ul><li>Does not change the language in documents or files.</li><li>You may need to restart some apps to see change.</li></ul>"
+        "tooltip": "<p>Change Language of Menus & Ribbons.</p><p>Works in Windows, Edge, Office and other programs with language packs installed.</p>",
+        "title": " Language of Menus",
```

Comment:
Is there a reason for having an extra space in the beginning?

Reply:
My mistake.